tests/lib/qunit/addons/composite/qunit-composite.js

```diff
@@ -39,6 +39,10 @@ function runSuite( suite ) {
 	}
 
 	QUnit.asyncTest( suite, function() {
+		// See: https://github.com/JamesMGreene/qunit-composite/issues/20
```

Comment:
It was required when I did my original testing, will confirm whether it's still needed now.